var/spack/repos/builtin/packages/py-sphinx/package.py

```diff
@@ -56,15 +58,17 @@ class PySphinx(PythonPackage):
     depends_on('py-jinja2@2.3:2', when='@:4.0.1', type=('build', 'run'))
     depends_on('py-pygments@2.0:', type=('build', 'run'))
     depends_on('py-docutils@0.14:0.17', when='@4:', type=('build', 'run'))
-    depends_on('py-docutils@0.12:', when='@:3', type=('build', 'run'))
+    depends_on('py-docutils@0.12:0.17', when='@:3', type=('build', 'run'))
```

Comment:
Old versions definitely don't support docutils 0.18 either, this causes the build to fail as 0.18 is chosen by default for any package that requires Sphinx 3.